internal/export/worker_test.go

```diff
@@ -148,220 +148,239 @@ func getKey(t *testing.T) []byte {
 func TestBatchExposures(t *testing.T) {
 	t.Parallel()
 
-	ctx := context.Background()
-	testDB, _ := testDatabaseInstance.NewDatabase(t)
-	testPublishDB := publishdb.New(testDB)
-
-	config := Config{
-		MinRecords:         1,
-		PaddingRange:       0,
-		MaxRecords:         1,
-		TruncateWindow:     time.Hour,
-		MaxInsertBatchSize: 100,
-	}
-	server := Server{
-		config: &config,
-		env:    serverenv.New(ctx, serverenv.WithDatabase(testDB)),
+	tests := []struct {
+		name  string
+		count int
+	}{
+		{
+			"base case",
```

Comment:
Nit: don't use spaces in test case names

Reply:
Done. Please add another lgtm